manifests/localenv/values.yaml

```diff
@@ -37,6 +37,8 @@ bq:
   service:
     type: NodePort
     httpNodePort: 31000
+  image:
+    repository: ghcr.io/ubisoft-potato/bigquery-emulator
```

Comment:
I have forked this repo to build and manage the BigQuery emulator by our self.
https://github.com/Ubisoft-potato/bigquery-emulator/pkgs/container/bigquery-emulator

Reply:
We could add workflow to do that(like devcontainer base image).
wdyt?

Reply:
Never mind. It's just an emulator. We can go as it is👍